fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -556,7 +556,9 @@ public KeysType getKeysType() {
     }
 
     public KeysType getKeysTypeByIndexId(long indexId) {
-        return indexIdToMeta.get(indexId).getKeysType();
+        MaterializedIndexMeta indexMeta = indexIdToMeta.get(indexId);
+        Preconditions.checkNotNull(indexMeta);
```

Comment:
You can add some message if you want to check it.

```suggestion
        Preconditions.checkNotNull(indexMeta, "index id: " + indexId);
```